### ISSUE TITLE
Using Chartkick instead of R?

### ISSUE THREAD

**Issue by gedankenstuecke** [openSNP/growthplots#3]
[Audy recommended Chartkick](http://chartkick.com/) on Gitter to avoid having R as an additional dependency. Could be worthwhile to do it?

**Comment by philippbayer** [comment]
We have some current JavaScript libraries loaded for the chartpies, we could replace everything with chartkick?

That would be much easier than having a second R-VM

**Comment by gedankenstuecke** [comment]
Currently we also have the Fitbit-data displayed using JS, which iirc is google charts, could replace that at the same time then.

**Comment by philippbayer** [comment]
Yes, unifying all plotting using one gem/JS-library would certainly increase loading times.. we could even get rid of jquery? I think that one just exists for the plotting

**Comment by gedankenstuecke** [comment]
Not sure whether we use jquery in other places for bootstrap in general maybe? But yeah, would be a good idea in any case. :+1:
